core-authentication/src/main/java/gg/solarmc/loader/authentication/LoginAttempt.java

```diff
@@ -66,8 +66,8 @@ public ResultType resultType() {
 	 *
 	 * @return the hashed password with instructions, or an empty optional if this is not {@code NEEDS_PASSWORD}
```

Comment:
you could change this to say a verifiable password:

Reply:
Right, will do